Makefile

```
@@ -662,27 +665,29 @@ docs:
 	fi
 	cd docs; make trans-copy clean build-offline;
 
-node_modules: package-lock.json
+.npmrc:
+	echo "audit=false fund=false package-lock=true save-exact=true " | tr " " "\n" > $@
+
+node_modules: package-lock.json .npmrc
 	npm install --no-save
 	@touch node_modules
 
 .PHONY: npm-cache
 npm-cache: .npm-cache $(FOMANTIC_WORK_DIR)/node_modules/fomantic-ui
 
-.npm-cache: package-lock.json
+.npm-cache: package-lock.json .npmrc
 	rm -rf .npm-cache
 	$(eval ESBUILD_VERSION := $(shell node -p "require('./package-lock.json').dependencies.esbuild.version"))
-	npm config --userconfig=.npmrc set cache=.npm-cache
+	$(eval ESBUILD_PLATFORMS := darwin-64 $(foreach arch,arm arm64 32 64,linux-${arch}) $(foreach arch,32 64,windows-${arch}))
```

Seriously just use the foreach again here to append the ESBUILD_VERSION

```suggestion
	$(eval ESBUILD_PLATFORMS := darwin-64 $(foreach arch,arm arm64 32 64,linux-${arch}) $(foreach arch,32 64,windows-${arch}))
	$(eval ESBUILD_BUILDS := $(foreach platform, $(ESBUILD_PLATFORMS), esbuild-@$(ESBUILD_VERSION)
```

---

As stated in OP, that'll mean these stuff gets output to the console. I'm not a fan of overly long lines either in source or log. Using `xargs` wouldn't have an impact on performance AFAICT.